adbms/src/main/scala/de/up/hpi/informationsystems/adbms/Dactor.scala

```diff
@@ -120,6 +122,12 @@ object Dactor {
       relDef -> RowRelation(relDef)
     ).toMap
 
+  def startSequentialFunction[S <: Request[_]](function: SequentialFunctionDef[S, _], context: ActorContext, sender: ActorRef)
```

Comment:
Can you use `ActorRefFactory` instead of `ActorContext` here? This would specify, why you need that parameter and also narrows down the functionality you can use (lower coupling).